app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::CommentsController < Api::V1::ApplicationController
+  def index
+    @post = Post.find(params[:post_id])
+
+    render json: @post.comments
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: @comment.errors, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:author_id, :post_id, :text)
+  end
```

Comment:
- [ ] kindly fix this error when creating a new comment 🙏 

![image](https://user-images.githubusercontent.com/89282221/218089689-5ed33e1b-1aa2-4437-8ead-0f3ec3f8c6f5.png)